src/main/java/com/deavensoft/paketomat/center/CenterServiceImpl.java

```diff
@@ -15,6 +18,8 @@
 public class CenterServiceImpl implements CenterService {
     private final CenterRepository centerRepository;
 
+    private PackageMapper packageMapper;
```

Comment:
Unused "private" fields should be removed

Reply:
Done

Reply:
Check again

Reply:
It is now deleted